src/department/department.entity.ts

```diff
@@ -22,4 +29,7 @@ export class DepartmentEntity {
 
   @UpdateDateColumn({ name: "updated_at" })
   updatedAt: Date;
+
+  @OneToMany(() => EmployeeEntity, (employee) => employee.department)
+  employees: EmployeeEntity[];
```

Comment:
this relation is ManyToMany
one department can belong to many employees and employee can be in multiple departments isn't it?

Reply:
One employee in only one department

Reply:
okay